src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import java.io.*;
```

Comment:
Ohh... You are not supposed to use wildcard. Try to import only the parts of java.io that is relevant to your project. As per Java Coding Standard. https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements